charts/rucio-server/templates/deployment.yaml

```diff
@@ -86,6 +91,24 @@ spec:
         secret:
           secretName: {{ .Release.Name }}-server-cafile
 {{- end }}
+{{- end }}
+{{- if .Values.policyPackages.enabled }}
+      initContainers:
+        - name: install-policy-packages
+          # A slim Python image could be used instead, but using the same image as the base image for the main container ensures that the same Python version is used
+          image: almalinux:9
+          command:
+            - "sh"
+            - "-c"
+            - |
```

Comment:
either add a `-x` or add logs in the 'else' case so that we know if no packages are already installed cause they already exist